lib/ansible/modules/cloud/ovirt/ovirt_snapshot.py

```diff
@@ -222,6 +236,23 @@ def restore_snapshot(module, vm_service, snapshots_service):
     }
 
 
+def remove_old_snapshosts(module, snapshots_service):
+    from datetime import datetime
+    import time
+    deleted_snapshot = []
+    changed = False
+    date_now = datetime.now()
+    for snapshot in snapshots_service.list():
+        if snapshot.vm and snapshot.vm.name == module.params.get('vm_name'):
+            diff = date_now - snapshot.date.replace(tzinfo=None)
+            if diff.days >= module.params.get('keep_days_old'):
+                snapshots_service.snapshot_service(snapshot.id).remove()
+                deleted_snapshot.append(get_dict_of_struct(snapshot))
+                changed = True
+                time.sleep(45)
```

Comment:
We need to wait until snaphsot is removed and vm ready to remove another snapshot, not any random wait time.